js/interface.js

```diff
@@ -2188,6 +2289,43 @@ $('#show-access-rules').click(function() {
   });
 });
 
+function getSecurityRule() {
```

Comment:
Missing JSDoc

Reply:
@squallstar added the JSDoc. its my bad.